### PR TITLE
Nar 1354 tech story: Restore queryBuilder Mock for unit tests

### PR DESCRIPTION
**Pull request by erik-gbg**

### CODE REVIEW

**Review by oskardrenske-ith — APPROVED** [review]

File: sokannonser/repository/__init__.py [hunk 1]
```diff
@@ -12,7 +11,16 @@
...
 log.info(f"Using synonym dictionary: {settings.ONTOLOGY_INDEX}")
 
-if settings.TEST_MODE:
-    elastic = None
-else:
-    elastic = create_elastic_client_with_retry(settings.ES_HOST, settings.ES_PORT, settings.ES_USER, settings.ES_PWD)
+
+class ElasticClient():
+    def __init__(self):
+         self.elastic = None
```
[the remark this answers is not in the document]
Comment (erik-gbg) [reply sokannonser/repository/__init__.py]: elastic is set to None when the package is imported order not to create an elastic client for the unit tests.
In the next step, when the elastic_client() function is called (from the API) the None tells us that it has not yet been created -- necessary in order to only creating it once.

**Comment by erik-gbg** [comment]
Oleg,

elastic_client() is supposed to be a function. Now you have changed it to a class object. And that doesn't work at all, of course.

**Comment by erik-gbg** [comment]
rebased onto develop head, to get rid of the bugs (when inspecting this branch)

**Review by halge — CHANGES_REQUESTED** [review]
i stop until "function" discussion is clear.

**Comment by erik-gbg** [comment]
Replaced by PR NAR-1354-the-one

**Pull request closed without merging**